internal/kafka/internal/migrations/20221215161330_add_kafka_promote_worker_in_leader_leases.go

```diff
@@ -0,0 +1,28 @@
+package migrations
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/db"
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+var promotingLeaderLeaseTypes = []string{"promoting_kafka"}
+
+func addKafkaPromoteWorkerInLeaderLeases() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "20221215161330",
+		Migrate: func(tx *gorm.DB) error {
+			for _, leaderLeaseType := range promotingLeaderLeaseTypes {
```

Comment:
Is there some reason we put it in an array? there does not seem to be a need for it